scripts/clean-db-dump.sql

```diff
@@ -0,0 +1,102 @@
+\echo 'Update users'
+UPDATE users
+SET name = 'Test user',
+    email_address = id || '@user.marketplace.team',
+    password = :bcrypt_password;
+
+-- Remove data about currently ongoing framework applications
+\echo 'Delete draft services for open frameworks'
+DELETE
+FROM draft_services
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+\echo 'Delete supplier frameworks for open frameworks'
+DELETE
+FROM supplier_frameworks
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+-- Remove data related to open DOS procurements
```

Comment:
I think using `CASCADE` is slightly more efficient for deleting child objects/avoiding orphans - so e.g. deleting draft `Brief` objects would also delete their child `BriefUser` objects without needing a separate statement. However it's obviously less explicit what's being deleted and what's not. If the script above has been serving us well in the past then I'm happy for it to continue 😄